interface_config.js

```diff
@@ -38,16 +38,14 @@ var interfaceConfig = {
     AUTHENTICATION_ENABLE: true,
 
     /**
-     * the toolbar buttons line is intentionally left in one line, to be able
-     * to easily override values or remove them using regex
+     * The name of the toolbar buttons to display in the toolbar. If present,
+     * the button will display. Exceptions are live streaming and recording
+     * which are configured in config.js. Also, the "profile" button will not
+     * display for user's with a jwt.
      */
     TOOLBAR_BUTTONS: [
-
-        // main toolbar
         'microphone', 'camera', 'desktop', 'fullscreen', 'fodeviceselection', 'hangup',
-
-        // extended toolbar
-        'profile', 'info', 'chat', 'recording', 'livestreaming', 'etherpad',
+        'profile', 'info', 'chat', 'etherpad',
```

Comment:
so recording / live streaming values don't show up in this array at all anymore?

Reply:
They could be present but their presence doesn't really do much. The buttons displaying is based on config alone. That is what you specified, no?

Reply:
i don't know the semantics of this array--i assumed a button _has_ to be here if it will be shown, so i just didn't know that wasn't the case.

Reply:
It doesn't have to be there with the current implementation. Take a look at toolbox.js, which has the logic to determine how to show the button, and the changes I made there. But what I remember being told is the the config is what will determine the display of the button. I somehow remember being told not to include the buttons anymore in the interfaceConfig value and instead rely on config only. I would prefer to keep the buttons in interfaceConfig for consistency, though. Please do clarify what you were relaying to me; these changes are trivial to do anyway.

Reply:
Talked with Brian in person. He doesn't recall the requirement of having interfaceConfig.TOOLBAR_BUTTONS cleared of recording and live streaming so I'm bringing them back to stay consistent with how all toolbar buttons appear in the array.